capstone.css

```diff
@@ -0,0 +1,362 @@
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  margin-left: -200px;
+}
+
+.lower-nav .active ul {
+  margin-left: -100px;
+}
+
+.lower-nav .information ul li {
+  display: flex;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.lower-nav .information ul li a {
+  text-decoration: none;
+  margin-top: 40px;
+  cursor: pointer;
+}
+
+.company {
+  margin-right: 60px;
+  display: none;
+}
+
+.main {
+  background-image: url(./images/backgroundM.png);
+}
+
+.main img {
+  width: 100%;
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hello {
+  display: flex;
+  flex-direction: column;
+  margin-left: 60px;
+}
+
+.hello .heading {
+  color: #ec5242;
+  margin-top: 140px;
+  font-size: 24px;
+  font-weight: 400;
+}
+
+.hello h2 {
+  color: #ec5242;
+  text-transform: uppercase;
+  font-family: "Cocogoose", sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+}
+
+.introduction {
+  width: 80%;
+  margin-top: -20px;
+  border: 2px solid #fff;
+  height: 80px;
+  margin-right: 20%;
+}
+
+.hall {
+  text-transform: capitalize;
+}
+
+.main-catagories {
+  background: #272a31;
+  background-image: url(images/tr-line.jpg);
+}
+
+.under-line {
+  display: inline-block;
+  margin-left: 50%;
+  margin-top: 24px;
+  color: #fff;
+  position: relative;
+  font-size: 24px;
+  font-weight: 700;
+  text-transform: capitalize;
+}
+
+.under-line::after {
+  content: "";
+  height: 2px;
+  width: 30%;
+  background: red;
+  position: absolute;
+  bottom: 0;
+  left: 30%;
+  margin-top: 4px;
+}
+
+.substation-design {
+  width: 95%;
+  display: flex;
+  flex-direction: row;
+  background: url(images/light.jpg);
+  border: 1px solid #fff;
+  margin-left: 20px;
+  margin-top: 12px;
+}
+
+.substation-design img {
+  width: 60%;
+  height: 40%;
+  margin-left: 20%;
+  margin-top: 50%;
+  margin-bottom: 50%;
+  align-items: center;
+}
+
+.substation-design h3 {
+  margin-left: 24px;
+  margin-top: 20%;
+  color: #ec5242;
+  align-items: center;
+}
+
+.see {
+  display: inline-block;
+  justify-content: center;
+  text-transform: uppercase;
+  position: relative;
+  margin-top: 60px;
+  margin-left: 40%;
+}
+
+.more {
+  color: #fff;
+}
+
+.see::after {
+  content: "";
+  height: 2px;
+  width: 100%;
+  background: #272a31;
+  position: absolute;
+  bottom: 0;
+  left: 0%;
+  margin-top: -120px;
+}
+
+.two {
+  margin-left: 24px;
+  background: #fff;
+}
+
+.two .under-line {
+  display: inline-block;
+  margin-left: 30%;
+  color: black;
+}
+
+.profesionalist {
+  display: flex;
+  flex-direction: column;
+}
+
+.profesionalist .title {
+  display: inline-block;
+  position: relative;
+  color: #ec5242;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.title::after {
+  content: "";
+  height: 2px;
+  width: 30%;
+  background: #272a31;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  margin-top: 4px;
+}
+
+.profesionalist .resume {
+  width: 80%;
+}
+
+.profesionalist .left {
+  margin-left: 24px;
+}
+
```

Comment:
- [ ] There are some elements that are not fully responsive at the mobile width. Kindly refer to the [behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and try to make it similar to the required design as much as possible possible.

## Your Design

![Electrical Engineering Portal (1)](https://user-images.githubusercontent.com/60618877/181589922-a8ce1819-d77a-4aaf-abf1-49340a7cff6b.gif)